vms/event/tests/test_shiftSignUp.py

```diff
@@ -32,6 +34,8 @@ def setUpClass(cls):
         super(ShiftSignUp, cls).setUpClass()
 
     def setUp(self):
+        from django.core.cache import cache
+        cache.clear()
```

Comment:
@rituranjna I am slightly confused :( Why are you clearing the cache here? And why only in this test file? I think there are around 17 test files